### PR TITLE
Update readme for explaining better the route order

### PR DESCRIPTION
**Pull request by allevo**
as titled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,7 +168,9 @@ Having a route with multiple parameters may affect negatively the performance, s
 
 <a name="match-order"></a>
 ##### Match order
-The routes are matched in the following order:
+The routing algorithm tokenizes the request path into segments.
```
Comment (mcollina) [inline README.md]: (divided by /).

Comment (allevo) [reply README.md]: => `The routing algorithm tokenizes (use the '/' to split) the request path into segments.`??

Comment (mcollina) [reply README.md]: not perfect but it’ll work.

**Comment by allevo** [comment]
Update as suggested

**Pull request merged**